modules/renter/persist.go

```diff
@@ -44,6 +44,17 @@ var (
 	shareVersion = "0.4"
 )
 
+type (
+	// persist contains all of the persistent renter data.
+	persistence struct {
+		MaxdownloadSpeed int64
+		MaxUploadSpeed   int64
+		StreamCacheSize  uint64
+		Repairing        map[string]string
```

Comment:
Don't need the repairing map, that's legacy code that's no longer necessary on master.